src/main/java/seedu/address/model/CommandRecord.java

```diff
@@ -27,16 +25,23 @@ public CommandRecord(Integer index, String commandString, CommandType commandTyp
         this.commandType = commandType;
     }
 
-    public CommandRecord(CommandType commandType) throws AlfredModelHistoryException {
-        if (commandType.equals(CommandType.END)) {
-            this.commandString = "*: Initialised State. Cannot undo.";
-            this.index = null;
-            this.commandType = CommandType.END;
-        } else {
-            throw new AlfredModelHistoryException("Previous commands cannot be properly initialised");
-        }
+    public static CommandRecord getUndoEndPoint() {
+        return new CommandRecord(null,
+                   "=============<< Cannot Undo Beyond This Point >>=============",
+                                 CommandType.END);
+    }
+
+    public static CommandRecord getRedoEndPoint() {
+        return new CommandRecord(null,
+                   "=============<< Cannot Redo Beyond This Point >>=============",
+                                 CommandType.END);
     }
 
+    public static CommandRecord getCurrentStatePoint() {
+        return new CommandRecord(null,
```

Comment:
Ahh I was planning to display the current command user is at, with the additional label "You are here", would that be clearer/less clear? :D

Reply:
I think you already display "You are here". That's cool as well. But is it okay to have the String value designated here instead? Because I think that's cleaner.

Reply:
ahhh I see, my concern was that the current command under 'You are here' won't be displayed as a simple name, I could add a new label above the below the current command to show "=====================<< Current State >>=====================". Or is just showing the border without simple name cleaner? hahaha  do update me on your choice 😄